test/runtests.jl

```diff
@@ -2,6 +2,23 @@ using Test, TimeSpans, Dates
 
 using TimeSpans: contains, nanoseconds_per_sample
 
+function naive_index_from_time(sample_rate, sample_time)
+    # This stepping computation is prone to roundoff error, so we'll work in high precision
+    sample_time_in_seconds = big(Dates.value(convert(Nanosecond, sample_time))) // big(TimeSpans.NS_IN_SEC)
```

Comment:
```suggestion
    sample_time_in_seconds = big(Dates.value(Nanosecond(sample_time))) // big(TimeSpans.NS_IN_SEC)
```
Does this actually have to be `convert` or will a constructor suffice?